### DISCUSSION TITLE
HX711 access from ESP32 Berry?

### DISCUSSION THREAD

**Discussion by mikep01** [arendst/Tasmota#15640] [Q&A] [answered]
Hi, is there a way to get at the current weight from berry with an HX711 connected to an ESP32?  Ideally the raw value but either will do.  I haven't been able to spot anything in docs - issuing a tasmota.cmd("status 8") shows the result in the berry console but I don't see how to capture & parse it.

**Comment by sfromis** [comment, marked as answer]
`var data = tasmota.cmd('Status 10')` (Status 8 gives the same) *already* captures and parses it.

With another sensor, I can then dive into the map hierarchy (parsed from JSON) doing things like:
`print(data['StatusSNS']['VEML7700']['Illuminance'])`

**Reply by sfromis** [reply]
You can also do it more directly, as `tasmota.read_sensors()` provides the JSON string, which you then can parse into a map hierarchy using 
```
import json
var data = json.load(tasmota.read_sensors())
print(data['VEML7700']['Illuminance'])
```
This method has less overhead, as there is no command execution or MQTT logging of the result.

**Reply by mikep01** [reply]
Thanks!  I was on the right track but made a syntax error and incorrectly assumed nothing was coming back from my tasmota.cmd.  This'll work well for me :).
